src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -1954,17 +1956,38 @@ static Set<String> expandFileNames(String[] filePathList, String queryId)
           result.add(((java.io.File) file).getCanonicalPath());
         }
       } catch (Exception ex) {
-        throw new SnowflakeSQLException(
-            queryId,
-            ex,
-            SqlState.DATA_EXCEPTION,
-            ErrorCode.FAIL_LIST_FILES.getMessageCode(),
-            "Exception: "
-                + ex.getMessage()
-                + ", Dir="
-                + entry.getKey()
-                + ", Patterns="
-                + entry.getValue().toString());
+        if (ex instanceof UncheckedIOException) {
+          if (!dir.exists()) {
+            logger.debug(
```

Comment:
I think we should add a warning here - for some users it may be unexpected

Reply:
Just to clarify, do you mean like a logger.warn level message, or just put "warning: ..." in the message body?

Reply:
I mean `logger.warn(...)`